cobalt/user_agent/BUILD.gn

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//testing/test.gni")
+
+static_library("user_agent") {
+  sources = [
+    "user_agent_platform_info.cc",
+    "user_agent_platform_info.h",
+  ]
+
+  public_deps = [
+    "//base",
+    "//cobalt:cobalt_build_info",
+    "//starboard/common",
+    "//v8:v8_version",
+  ]
+}
+
+test("user_agent_unittests") {
```

Comment:
This will create an executable `user_agent_unittests`, and what we want is a single `cobalt_unittests`. 

So this must be either a `component` or a `static_library`, and be added to the `deps` of `cobalt_unittests` in cobalt/testing/BUILD.gn.